profile_library/ROBB/ROB_200-017-0/model.json

```diff
@@ -0,0 +1,17 @@
+{
+  "measure_description": "Manually measured average of 2 plugs",
+  "measure_method": "manual",
+  "measure_device": "Brennenstuhl PM 231 E",
+  "name": "ROBB Zigbee Smart Plug",
+  "standby_power": 0.2,
+  "sensor_config": {
+    "power_sensor_naming": "{} Device Power",
+    "energy_sensor_naming": "{} Device Energy"
+  },
+  "device_type": "smart_switch",
+  "calculation_strategy": "fixed",
+  "fixed_config": {
+    "power": 0.65
+  },
+  "created_at": "2024-05-07T20:30:00"
```

Comment:
```suggestion
  "created_at": "2024-08-12T08:00:00"
```

Reply:
I actually created the profile in May, but forgot to send a PR. Propose to leave as is.